cmd/dep/status.go

```diff
@@ -49,6 +55,7 @@ func (cmd *statusCommand) Register(fs *flag.FlagSet) {
 	fs.BoolVar(&cmd.detailed, "detailed", false, "report more detailed status")
 	fs.BoolVar(&cmd.json, "json", false, "output in JSON format")
 	fs.StringVar(&cmd.template, "f", "", "output in text/template format")
+	fs.StringVar(&cmd.output, "o", "output.svg", "output file")
```

Comment:
It is necessary - we're not printing dot output in the default case. The parameter is needed to indicate that dot output is desired. (So, a `bool` instead of a `string`)